docs/network-flow-visibility.md

```diff
@@ -696,11 +696,17 @@ a new dashboard, please follow this [doc](https://grafana.com/docs/grafana/lates
 on how to build a dashboard.
 
 By clicking on the "Save dashboard" button in the Grafana UI, the changes to the
-dashboards will be persisted in the Grafana database at runtime, but they will be
-lost after restarting the Grafana deployment. To restore those changes after a restart,
-as the first step, you will need to export the dashboard JSON file following the
-[doc](https://grafana.com/docs/grafana/latest/dashboards/export-import/), then there
-are two ways to import the dashboard depending on your needs:
+dashboards will be saved in the Grafana database.
```

Comment:
This paragraph can be removed, with its content combined with the next paragraph.